### PR TITLE
fix: use patched version for vulnerable league/flysystem repository

### PR DESCRIPTION
**Pull request by bartlomiejmarszal**
FIx Time-of-check Time-of-use (TOCTOU) Race Condition in league/flysystem

### CODE REVIEW

**Review by julien-sebire — APPROVED** [review]
Go for it Bart!

**Comment by Karol-Stelmaczonek** [comment]
No composer.lock file changed?

**Comment by andreluizmachado** [comment]
There is no composer lock on this repo @Karol-Stelmaczonek

File: composer.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
 		"ext-libxml": "*",
 		"ext-mbstring": "*",
 		"ext-simplexml": "*",
-		"league/flysystem": "^1.0|^2.0|^3.0",
+		"league/flysystem": "^1.0|2.1.1|^3.0",
```
Comment (andreluizmachado) [inline composer.json:34]: Why did you restricted tho this version???
Why not ^2.1 or ~2.1.1 if you do not want to upgrade the features?

**Review by Karol-Stelmaczonek — APPROVED** [review]

**Pull request merged**